molecule/default/tests/test_default.py

```diff
@@ -12,7 +12,12 @@
 ).get_hosts("all")
 
 
-@pytest.mark.parametrize("x", [True])
-def test_packages(host, x):
-    """Run a dummy test, just to show what one would look like."""
-    assert x
+@pytest.mark.parametrize("f", ["/etc/ssh/sshd_config.d/99-allow-rsakeys.conf"])
+def test_files(host, f):
```

Comment:
Just as a thought should this verify that the two keywords are there?

Reply:
Please see commit ded9875, in which I verify that the correct version of the `PubkeyAccepted*` keyword is there.  I don't think it makes sense to check for the other keyword, since that would simply be testing the `ansible.builtin.copy` module.

Reply:
Can this conversation be resolved?